File: lame_king/lame_king.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int case_num;
    scanf("%d", &case_num);
    for (int i = 0; i < case_num; i++)
    {
        int a, b;
        scanf("%d %d", &a, &b);
        int num_secs;
        if (abs(a) > abs(b))
        {
            num_secs = (2 * abs(a)) - 1;
        }

        else if (abs(b) > abs(a))
        {
            num_secs = (2 * abs(b)) - 1;
        }

        else
        {
            num_secs = 2 * abs(a);
        }

        printf("%d\n", num_secs);
    }
}

File: lame_king/lame_king.py
def main():
    case_num = int(input())
    for _ in range(case_num):
        a, b = input().split()
        a  = int(a)
        b = int(b)
        if abs(a) > abs(b):
            num_secs = 2 * abs(a) - 1
        elif abs(b) > abs(a):
            num_secs =  2 * abs(b) - 1
        else:
            num_secs = 2 * abs(a)
        print(num_secs)
    
    
    
    
    
    
    
    
    
    
    
    
    
if __name__ == "__main__":
    main()